### DISCUSSION TITLE
🚀 Exciting Update: GPIOViewer Arduino Library v1.5!

### DISCUSSION THREAD

**Discussion by thelastoutpostworkshop** [thelastoutpostworkshop/gpio_viewer#90] [Announcements]
I am thrilled to announce the latest release of the GPIOViewer Arduino Library, version 1.5, 
now integrated with **a brand-new web application!** 

This update marks a significant leap forward, paving the way for an even more user-friendly experience. 
Get ready to explore enhanced features and seamless additions of new functionalities. 

You can are welcomed to contribute to the new [open source Vuejs web application](https://github.com/thelastoutpostworkshop/GPIOViewerVueApplication).

Report any issues [here](https://github.com/thelastoutpostworkshop/GPIOViewerVueApplication/issues).

**Comment by MakerBits-co** [comment]
estaba muy emocionado, pero lamentablemente no me funciona, se reinicia y muestra error:  assert failed: tcp_alloc /IDF/components/lwip/lwip/src/core/tcp.c:1851 (Required to lock TCPIP core functionality!)
